src/index.html

```diff
@@ -11,7 +11,132 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="http://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post">
+
+      <fieldset class="groups">
+        <legend>Personal information</legend>
+
+        <label>
+          <div class="inputs">
+            Surname:  <input name="surname" type="text" autocomplete="off" required>
```

Comment:
Please, apply the same changes to all other elements which have 3 and more attributes
```suggestion
            Surname:  
            <input 
              name="surname" 
              type="text" 
              autocomplete="off" 
              required
            >
```